static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,30 @@ define([
         }
     }
 
+    function paidforCardLoaded(el) {
+        var height,
+            $adSlot = $(el);
```

Comment:
Does it mean that I have to use that approach? I am used to one var and I think it's cleaner so I prefer to stay with it.